client/src/api.ts

```diff
@@ -50,17 +51,22 @@ export const pullNewGame = async (playerNumber: number): Promise<GameState> => {
   }
 };
 
-export const pullJoinGame = async (idGame: number): Promise<GameState> => {
+export const pullJoinGame = async (
+  idGame: number,
+  playerId: number,
+): Promise<GameState> => {
   try {
-    const response = await fetch(`${URL}/games/${idGame}/join`, {
+    let response = await fetch(`${URL}/games/${idGame}/join`, {
       method: 'PUT',
+      body: JSON.stringify({ playerId }),
       headers: getHeaders(),
     });
+    response = status(response);
```

Comment:
I do not understand the meaning of this function

Reply:
This is to check if the answer is an error. 
We need to rewrite our entire error system to the client side.